File: src/dao/dto/DataLogDto.kt
package cn.dairo.cls.dao.dto

/**
 * 流量统计表DTO
 */
class DataLogDto {

    /**
     * 目标id
     */
    var targetId: Int? = null

    /**
     * 年
     */
    var year: Long? = null

    /**
     * 年月
     */
    var ym: Long? = null

    /**
     * 年月日
     */
    var ymd: Long? = null

    /**
     * 年月日时
     */
    var ymdh: Long? = null

    /**
     * 年月日时分
     */
    var ymdhm: Long? = null

    /**
     * 入网流量
     */
    var inData: Long? = null

    /**
     * 出网流量
     */
    var outData: Long? = null

    /**
     * 统计类型, 1:隧道  2:数据转发
     */
    var type: Int? = null
}

File: dao/dto/DataLogDto.go
package dto

// 流量统计表DTO
type DataLogDto struct {

	//目标id
	TargetId int

	//年
	Year int64

	//年月
	Ym int64

	//年月日
	Ymd int64

	//年月日时
	Ymdh int64

	//年月日时分
	Ymdhm int64

	//入网流量
	InData int64

	//出网流量
	OutData int64

	//统计类型, 1:隧道  2:数据转发
	Mode int
}
